bundle/deploy/terraform/import.go

```diff
@@ -0,0 +1,129 @@
+package terraform
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/databricks/cli/bundle"
+	"github.com/databricks/cli/libs/cmdio"
+	"github.com/databricks/cli/libs/log"
+	"github.com/hashicorp/terraform-exec/tfexec"
+)
+
+type BindOptions struct {
+	AutoApprove  bool
+	ResourceType string
+	ResourceKey  string
+	ResourceId   string
+}
+
+type importResource struct {
+	opts *BindOptions
+}
+
+// Apply implements bundle.Mutator.
+func (m *importResource) Apply(ctx context.Context, b *bundle.Bundle) error {
+	tf := b.Terraform
+	if tf == nil {
+		return fmt.Errorf("terraform not initialized")
+	}
+
+	err := tf.Init(ctx, tfexec.Upgrade(true))
+	if err != nil {
+		return fmt.Errorf("terraform init: %w", err)
+	}
+
+	importsFilePath, err := m.writeImportsFile(ctx, b)
+	if err != nil {
+		return fmt.Errorf("write imports file: %w", err)
+	}
+	log.Debugf(ctx, "imports.tf file written to %s", importsFilePath)
+
+	buf := bytes.NewBuffer(nil)
+	tf.SetStdout(buf)
+	changed, err := tf.Plan(ctx)
+	if err != nil {
+		return fmt.Errorf("terraform plan: %w", err)
+	}
+
+	if changed && !m.opts.AutoApprove {
+		cmdio.LogString(ctx, buf.String())
+		ans, err := cmdio.AskYesOrNo(ctx, "Confirm import changes? Changes will be remotely only after running 'bundle deploy'.")
+		if err != nil {
+			return err
```

Comment:
The file isn't removed here. There must be no case where something happens and on the second call to deploy the resource is imported anyway. Could leverage defer?